stable/defaultstorageclass/templates/deployment.yaml

```
@@ -4,19 +4,22 @@ kind: Deployment
 metadata:
   creationTimestamp: null
   labels:
+    app: dstorageclass-controller-manager  
```

What do you think about using this instead. If all our charts did it this way, it would make human interaction easier because the resources would have predictable labels.
```suggestion
    app: defaultstorageclass
```